index.js

```diff
@@ -18,45 +14,32 @@ function getProcess() {
 }
 
 function terminalSize() {
-  return new IO(getProcess)
-    .map(R.prop('stdout'))
-    .map((outputStream) => ({
-      width: outputStream.columns,
-      height: outputStream.rows,
-    }))
+  return new IO(getProcess).map(R.prop('stdout')).map((outputStream) => ({
```

Comment:
Old style (each call on each line) is more readable

Reply:
I will fix. Prettifier too :/